dexcom-analysis.py

```diff
@@ -0,0 +1,194 @@
+# import os
+import ctypes
+import pandas as pd
+# import matplotlib.pyplot as plt
+import altair as alt
+import webview
+
+# Get screen resolution
+user32 = ctypes.windll.user32
```

Comment:
This code only seems to work on Windows.. but I see `ctypes` does have MacOS and Linux variants. 
Is it possible to have this be dynamic based on the user's operating system?

Reply:
I changed the way to get the screen resolution. Using `pyautogui` package instead of `ctypes`.